lib/round.rb

```diff
@@ -0,0 +1,56 @@
+# require './lib/turn'
+
+class Round
+  attr_reader :turns, :number_correct
+  attr_accessor :deck
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+  end
```

Comment:
since you don't need to access `number_correct` outside of this class, we don't need to set it as an attr_reader. Keep it as an instance variable in your initialize method though so you can reference it within this class in any method.